pkg/cmd/roachtest/roachtestutil/mixedversion/helper.go

```diff
@@ -72,10 +72,17 @@ type (
 	}
 )
 
+// Connect returns a connection pool to the given node. Note that
```

Comment:
nit: i think it would be clearer to state that `Connect` returns a connection _from a_ pool, not a "connection pool", right? 

Same comment applies below.

Reply:
`*sql.DB` is a connection pool. `*sql.Conn` is a single connection. Most existing functions in roachtest (and elsewhere) were written with the assumption that you pass the former. It's also the recommended approach:

> Prefer running queries from [DB](https://pkg.go.dev/database/sql#DB) unless there is a specific need for a continuous single database connection.

https://pkg.go.dev/database/sql#Conn

Reply:
Heh, true, I think that's common in our codebase in general. Roachtest's `c.Conn()` is the same.